### DISCUSSION TITLE
Best configs for algorithms

### DISCUSSION THREAD

**Discussion by Bombex** [mikel-brostrom/yolo_tracking#1259] [Q&A] [answered]
Hello @mikel-brostrom and team. Firstly thank you for this amazing repository.

The boxmot/configs folder doesn't have the best hyperparameters for tracking algorithms, does it?  

If not, how can we get better parameters with better performance in the benchmarks? 

And maybe you have steps or advice on what we need to do to implement these algorithms for our data, in my case for an RTSP stream?

**Comment by mikel-brostrom** [comment, marked as answer]
> The boxmot/configs folder doesn't have the best hyperparameters for tracking algorithms, does it?

Whatever parameters you may find under `config` won't be the optimal ones for you data

> If not, how can we get better parameters with better performance in the benchmarks?

Run `examples/evolve.py` on your data

**Reply by Bombex** [reply]
What data do I need? I only have a video stream. Do I need to download the video or take images from the video and annotate it as one of the benchmarks?

**Reply by mikel-brostrom** [reply]
If you don't have grund truth for any example video you cannot automatically fine-tune the paramerers and would need to rely on manual tuning

**Reply by mikel-brostrom** [reply]
You can check out MOT17-mini for the structure and data format needed
